components/ILIAS/User/classes/class.ilUserImportParser.php

```diff
@@ -497,7 +497,6 @@ public function importBeginTag(
                         case 'local':
                         case 'shibboleth':
                         case 'script':
-                        case 'cas':
```

Comment:
Hi @kergomard  , we had to remove `cas` from the import parser to completely remove the CAS authentication from our code base. If you have any objections, please leave a comment.